src/main/kotlin/com/beust/kobalt/maven/PomGenerator.kt

```diff
@@ -55,7 +52,7 @@ public class PomGenerator @Inject constructor(@Assisted val project: Project) {
 
         val buildDir = KFiles.makeDir(project.directory, project.buildDirectory!!)
         val outputDir = KFiles.makeDir(buildDir.path, "libs")
-        val mavenId = MavenId.create(project.group!!, project.artifactId!!, project.packaging, project.version!!)
+        val mavenId = MavenId(project.group!!, project.artifactId!!, project.version!!, project.packaging)
```

Comment:
Wouldn't it make more sense to keep the arguments of `MavenId` in the same order they appear in the version?

`com.android:x:aar:1.0:packaging` means the order should be `groupId:artifactId:packaging:version:packaging`